From c71c17951e5f00195c9cacd6829a359c3d253714 Mon Sep 17 00:00:00 2001
From: Ren Yu <yux.ren@intel.com>
Date: Mon, 29 Jul 2019 10:51:12 +0800
Subject: [PATCH] Add PreInterruptFlag properity in DBUS.

PreTimeoutInterruptOccurFlag in DBUS would be set 'true'
when watchdog pre-timeout interrupt occurred.

Tested:
Enable command(raw 0x06 0x31) that get message flag
can set right bit about watchdog,
need record PreTimeoutInterruptOccurFlag
at xyz.openbmmc_project.State.Watchdog when watchdog
pre-timeout interrupt occurred.

Signed-off-by: Ren Yu <yux.ren@intel.com>
---
 yaml/xyz/openbmc_project/State/Watchdog.interface.yaml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml b/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml
index a67a6a30ea2c..d1ae8e3411e6 100644
--- a/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml
+++ b/yaml/xyz/openbmc_project/State/Watchdog.interface.yaml
@@ -59,6 +59,11 @@ properties:
       description: >
           The timer user at the time of expiration.
       default: 'Reserved'
+    - name: PreTimeoutInterruptOccurFlag
+      type: boolean
+      description: >
+          PreTimeoutInterruptOccurFlag that preTimeoutInterrupt action occurred.
+      default: false
 
 enumerations:
    - name: Action
-- 
2.17.1

